City-of-Helsinki/helsinki-paatokset
# Pp 201 trustee page

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-201

**How to test**

Checkout branch and run `drush cim -y; drush cr` and make sure you have trustee nodes.

Go to some trustee page and add data to the Drupal fields. If the trustee doesn't have any initiatives or resolutions, add some.

Check that the layout matches with the design and that the fields render nicely if there is data or if there isn't data. 

The content in "Puheenvuorot" is dummy data since the data is supposed to come from datapumppu integration.
